pandas/tests/test_categorical.py

```diff
@@ -57,6 +57,24 @@ def test_getitem_listlike(self):
         expected = c[np.array([100000]).astype(np.int64)].codes
         tm.assert_numpy_array_equal(result, expected)
 
+    @pytest.mark.parametrize("method",
```

Comment:
not sure if this is passing linting I would write this like

```
@pytest.mark.parametrize(
    "method",
    [
        lambda .....
    ])
```

to get more indent